src/client/client4.js

```diff
@@ -523,9 +523,8 @@ export default class Client4 {
 
     getProfilesInChannel = async (channelId, page = 0, perPage = PER_PAGE_DEFAULT) => {
         this.trackEvent('api', 'api_profiles_get_in_channel', {channel_id: channelId});
-
         return this.doFetch(
-            `${this.getUsersRoute()}${buildQueryString({in_channel: channelId, page, per_page: perPage})}`,
+            `${this.getUsersRoute()}${buildQueryString({in_channel: channelId, page, per_page: perPage, sort: 'status'})}`,
```

Comment:
Why isn't this optional? what happens to those that are using an older server?

Reply:
What do you mean by optional?
Concerning supporting other server versions, I assume the server and the webapp would be up to date with each other; however I now realize that would screw up the mobile app.

So, how do I detect the server version to decide if this attribute is included or not? Or what is the strategy for backwards compatibility here?